tech.json

```diff
@@ -2552,6 +2552,63 @@
                 "This tech should not be used directly; instead use a helper,",
                 "in order to correctly account for the amount of Crystal Flash refill, which depends on the environment."
               ]
+            },
+            {
+              "name": "canXModeBlueSuit",
+              "techRequires": [
+                "canXMode",
+                "canCarryBlueSuit"
+              ],
+              "otherRequires": [
+                {"gainBlueSuit": {}},
+                {"noFlashSuit": {}}
+              ],
+              "note": [
+                "The ability to gain a blue suit by shinesparking while in X-Mode and exiting X-Mode before the shinespark has crashed.",
+                "Release dash during the shinespark wind-up animation to exit X-Mode, interrupting the shinespark and gaining a blue suit.",
+                "A blue suit will not be gained if dash is released after a shinespark has crashed against a wall or object.",
+                "Releasing dash mid shinespark will cause Samus to SuperJump and potentially go out of bounds but this can also be used to jump away from spikes or thorns.",
+                "Care must be taken around spikes or thorns after a blue suit is gained if there are active enemies as I-frames will expire quickly and escaping can be difficult."
+              ]
+            },
+            {
+              "name": "canDoubleXModeBlueSuit",
+              "techRequires": [
+               "canXModeBlueSuit",
+               "canBeVeryPatient",
```

Comment:
```suggestion
```
The maneuver isn't slow to pull of, so Patience would only mean lenience.  But with techs that is handled by tech difficulty placement. 
The usage of the tech may have lenience on each of the XModes separately, or it could focus more on the final XMode blue suit by assuming the player has practiced the trick enough to get it semi reliable.  
Then if there is a harder than normal instance of `canDoubleXModeBlueSuit`, it would either be a `notable` or gain `canBeExtremelyPatient`.  Since this would be placed in insane already.